schemaorg.json

```diff
@@ -0,0 +1,100 @@
+{
+  "@context": "http://schema.org",
+  "@graph": [
+    {
+      "type": "SoftwareSourceCode",
+      "author": [
+        {
```

Comment:
Also, If you are referencing by id, it is better to list the `id` purely, e.g. it should be `"author": ["https://orcid.org/0000-0002-2815-0399", "https://orcid.org/0000-0002-1642-628X"]`, and no `"id":`.  This makes it clear that it is a reference, because the child object is list of URIs (references), not a list of child nodes (stuff with `{}`).